cmd/bisync/log.go

```diff
@@ -29,14 +29,23 @@ func (b *bisyncRun) indent(tag, file, msg string) {
 		logf = fs.Logf
 	}
 
+	if b.opt.DryRun {
+		logf = fs.Logf
+	}
+
 	if tag == "Path1" {
 		tag = Color(terminal.CyanFg, "Path1")
+		msg = strings.Replace(msg, "Queue copy to", Color(terminal.GreenFg, "Queue copy to Path2"), -1)
+		msg = strings.Replace(msg, "Queue delete", Color(terminal.RedFg, "Queue delete from Path1"), -1)
+	} else if tag == "Path2" {
+		tag = Color(terminal.GreenFg, "Path2")
```

Comment:
We currently try to use green/yellow/red for their universal meanings of new/modified/deleted or success/warning/error, so I would choose a different color (or leave the default of `BlueFg`).

It should also avoid `CyanFg` (already used for Path1) and `MagentaFg` (already used as the default `msg` color). So that is why `BlueFg` was one of the only options left 🙂